Generate/src/Medication-9-0-7/XIS-Server/DispenseRequest/mp9-vv-fhir3-0-2-xis-7-1.xml

```diff
@@ -11,21 +11,14 @@
       <name value="Scenario 7.1"/>
       <description value="Scenario 7.1 - Alle ingrediënten"/>
       <nts:include value="xis-ma-vv-operation-search">
-         <nts:with-parameter name="description"
-                       value="Test XIS server to serve DispenseRequest (NL: VerstrekkingsVerzoek) - Alle ingrediënten"/>
+         <nts:with-parameter name="description" value="Test XIS server to serve DispenseRequest (NL: VerstrekkingsVerzoek) - Alle ingrediënten"/>
          <nts:with-parameter name="accept" value="xml"/>
-         <nts:with-parameter name="params"
-                       value="?category=http://snomed.info/sct|52711000146108&amp;_include=MedicationRequest:medication"/>
+         <nts:with-parameter name="params" value="?category=http://snomed.info/sct|52711000146108&amp;_include=MedicationRequest:medication"/>
       </nts:include>
       <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
       <nts:include value="assert-responseBundleContent" scope="common">
          <nts:with-parameter name="warningOnly" value="true"/>
       </nts:include>
-      <nts:include value="assert-returnCount" scope="project">
-         <nts:with-parameter name="description"
-                       value="Confirm that the returned searchset Bundle contains 3 DispenseRequest."/>
-         <nts:with-parameter name="expression"
-                       value="Bundle.entry.where(resource.is(MedicationRequest)).count() = 3"/>
-      </nts:include>
+      <nts:include value="assert-returnCount-at-least-one-vv.xml" scope="project"/> 
```

Comment:
Klopt het dat hier count() = 3 wordt vervangen door >= 1? Dan klopte 3 dus uberhaupt niet

Reply:
Yes! De testscripts moeten individueel van elkaar kunnen worden uitgevoerd. Men hoeft niet alle 7-1 7-2 en 7-3 te doen. Maar zou ook 1 of 2 van de 3 kunnen doen.